ui/perfherder/alerts/SelectAlertsDropdown.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DropdownItem from 'reactstrap/lib/DropdownItem';
+import DropdownMenu from 'reactstrap/lib/DropdownMenu';
+import DropdownToggle from 'reactstrap/lib/DropdownToggle';
+import UncontrolledDropdown from 'reactstrap/lib/UncontrolledDropdown';
+import Input from 'reactstrap/lib/Input';
+import Label from 'reactstrap/lib/Label';
+
+import { alertStatus as status } from '../constants';
+
+class SelectAlertsDropdown extends React.PureComponent {
+  render() {
+    const {
+      selectAlertsByStatus,
+      user,
+      allSelected,
+      alertSummary,
+    } = this.props;
+    const { untriaged, triaged, all, none } = status;
+
+    return (
+      <React.Fragment>
+        <Label check className="pl-1">
+          <Input
+            data-testid={`alert summary ${alertSummary.id.toString()} checkbox`}
```

Comment:
I prefer we don't use `data-testid` on the frontend just to enhance our testing abilities.
react testing library has lots of ways of identifying test components. For example, by selecting them based on ARIA attributes.

Reply:
Seems like without this, we'd break some of our existing tests. I'm cancelling this request.